### PR TITLE
Do not extrapolate initial rs and rv values in the depth tables

### PR DESCRIPTION
**Pull request by totto82**
Makes initial rs values when outside depth range compatible with Ecl.

### CODE REVIEW

File: opm/core/utility/linearInterpolation.hpp [hunk 1]
```diff
@@ -70,6 +70,22 @@ namespace Opm
 	return  (yv[ix2] - yv[ix1])/(xv[ix2] - xv[ix1])*(x - xv[ix1]) + yv[ix1];
     }
 
+    inline double linearInterpolationNoExtrapolation(const std::vector<double>& xv,
+                                      const std::vector<double>& yv, double x)
+    {
+        int ix1 = tableIndex(xv, x);
```
Comment (atgeirr) [inline opm/core/utility/linearInterpolation.hpp]: Please move these definitions below the end-value tests.

File: opm/core/utility/linearInterpolation.hpp [hunk 2]
```diff
@@ -70,6 +70,22 @@ namespace Opm
 	return  (yv[ix2] - yv[ix1])/(xv[ix2] - xv[ix1])*(x - xv[ix1]) + yv[ix1];
     }
 
+    inline double linearInterpolationNoExtrapolation(const std::vector<double>& xv,
+                                      const std::vector<double>& yv, double x)
```
Comment (atgeirr) [inline opm/core/utility/linearInterpolation.hpp]: Weird indentation, please fix.

**Comment by totto82** [comment]
Fixed. And yes it is tested on model 2 and norne.

**Comment by atgeirr** [comment]
I don't really understand the Travis error, so I'll run Jenkins to check this.

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
Thanks for the patch, merging as it is green and you already tested it.

**Pull request merged**